docs/support/faq.md

```diff
@@ -58,44 +66,58 @@ You can also perform the following to verify the media item has a GUID Overseerr
    3. TheTVDB agent `guid="com.plexapp.agents.thetvdb://78874/1/1"`
    4. Legacy Plex Movie agent `guid="com.plexapp.agents.imdb://tt0765446"`
 
-### TV series requests are failing after I updated Overseerr!
-
-**A:** Language profile support for Sonarr was added in [#860](https://github.com/sct/overseerr/pull/860), along with a new "Language Profile" required setting. If your TV series requests are failing, please make sure that you have a default language profile configured for each of your Sonarr servers in **Settings &rarr; Services**.
-
 ### Where can I find the logs?
 
-**A:** The logs are located at `<Overseeerr-install-directory>/logs/overseerr.log`
+Log files are located in `<Overseeerr config directory>/logs/`.
```

Comment:
Can we leave `overseerr.log` in there? Maybe lets just link to asking for support page so we only keep this in one place? https://docs.overseerr.dev/support/asking-for-support#how-can-i-share-my-logs

Reply:
Fixed!